superset-frontend/src/logger/LogUtils.ts

```diff
@@ -59,8 +59,14 @@ export const LOG_EVENT_TYPE_USER = new Set([
 ]);
 
 export const Logger = {
-  // note that this returns ms since page load, NOT ms since epoch
+  softNavigationStart: 0,
+
+  resetStartTime() {
+    this.softNavigationStart = window.performance.now();
+  },
```

Comment:
The `window.performance` refers the start time as ["time origin"](https://developer.mozilla.org/en-US/docs/Web/API/DOMHighResTimeStamp#the_time_origin), so how about renaming the variables like:

```ts
  markTimeOrigin() {
    this.timeOriginOffset = window.performance.now();
  }

  getTimestamp() {
    return Math.round(window.performance.now() - this.timeOriginOffset);
  },
```